infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/server/secure/jwtproxy/JwtProxyConfigBuilder.java

```diff
@@ -36,27 +44,42 @@
  * @author Sergii Leshchenko
  */
 public class JwtProxyConfigBuilder {
-  private final List<VerifierProxy> verifierProxies = new ArrayList<>();
   private final String workspaceId;
+  private final URI authPageUrl;
+  private final String issuer;
+  private final String ttl;
+  private final List<VerifierProxy> verifierProxies = new ArrayList<>();
   private static final ObjectMapper YAML_PARSER =
```

Comment:
Please place a static field above non-static field and separate it with new line.